hive/src/main/java/com/netflix/iceberg/hive/HiveTableOperations.java

```diff
@@ -57,6 +57,9 @@
 /**
  * TODO we should be able to extract some more commonalities to BaseMetastoreTableOperations to
  * avoid code duplication between this class and Metacat Tables.
+ *
+ * Note! This class is not thread-safe as {@link ThriftHiveMetastore.Client} does not behave
```

Comment:
Tables should be thread-safe. Can you open an issue to fix this by using a thread-local client or a pool?

Reply:
Makes sense. Shall we remove this note from the PR now?

Reply:
No, this can stay in until we have a fix for it. Good to note it.